examples/anthropic/search_link_graph_haiku.py

```diff
@@ -34,22 +34,22 @@
 }
 
 # ************************************************
-# Create the SearchGraph instance and run it
+# Create the SearchLinkGraph instance and run it
 # ************************************************
 
-search_graph = SearchGraph(
+search_link_graph = SearchLinkGraph(
```

Comment:
you have correctly changed all examples with the search link graph, which previously were not aligned, and used wrong classes, but all examples are broken cause the search link graph takes a URL source as input not a prompt